src/main/connector/hardware/Hardware/Hardware.yaml

```diff
@@ -188,6 +188,11 @@ metrics:
     type: Gauge
     unit: By
 
+  hw.memory.utilization:
```

Comment:
`hw.memory` is supposed to represent memory modules. As such, there is no concept of "utilization" or "free" in a module.

If we want to represent the memory usage, it's always handled by the operating system and is represented with the `system.memory.usage` and `system.memory.utilization` metrics.

Reply:
Yes, I had the same discussion with Thomas about system/hardware memory utilization. I don't know if the OS is accessible in Netscaler to report memory utilization.

Reply:
No matter whether we access the actual OS or not, the memory usage will be reported with `system.memory.usage`, and CPU utilization with `system.cpu.utilization` (and corresponding metrics).

This will apply to any kind of system: network devices, storage systems, controllers, etc. 😊

Reply:
Ok let's collect system.memory.utilization then ;)